Integrations/FortiGate/FortiGate.yml

```diff
@@ -437,6 +437,40 @@ script:
       description: Was address group deleted
       type: boolean
     description: Delete an address group from FortiGate firewall
+  - name: fortigate-ban-ip
+    arguments:
+    - name: ip_addresses
+      required: true
+      description: List of IP Addresses to ban, comma separated. IPv4 and IPv6 addresses
+        are allowed.
+      isArray: true
+    - name: expiry
+      description: Time until expiry in seconds. 0 for indefinite ban.
+    description: Immediately add one or more users to the banned list.
+  - name: fortigate-unban-ip
+    arguments:
+    - name: ip_addresses
```

Comment:
our convention is singular (the user can understand from the description that CSV is supported), please change to `ip_address`